antsibull/build_ansible_commands.py

```diff
@@ -98,39 +123,47 @@ def write_setup(ansible_version, ansible_base_version, collection_deps, package_
         f.write(setup_contents)
 
 
-def write_python_build_files(ansible_version, ansible_base_version, collection_deps, package_dir,
-                             release_notes: t.Optional[ReleaseNotes] = None, debian: bool = False):
+def write_python_build_files(ansible_version: PypiVer,
+                             ansible_base_version: PypiVer,
+                             collection_deps: str,
+                             package_dir: str,
+                             release_notes: t.Optional[ReleaseNotes] = None,
+                             debian: bool = False) -> None:
     copy_boilerplate_files(package_dir)
     write_manifest(package_dir, release_notes, debian)
     write_setup(ansible_version, ansible_base_version, collection_deps, package_dir)
 
 
-def write_debian_directory(ansible_version, package_dir):
+def write_debian_directory(ansible_version_version: str, package_dir: str) -> None:
```

Comment:
Why the rename?

Reply:
I think that is a rebase artefact. I was wondering about that as well, and first thought that it came from your code, but I guess I introduced it during rebasing after #139 was merged.

Reply:
That's gone now.